openedx/features/caliper_tracking/transformers/team_transformers.py

```diff
@@ -112,3 +112,52 @@ def edx_team_learner_added(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_team_activity_updated(current_event, caliper_event):
+    """
+    When team discussion activity has occurred on a team,
+    including a team member posting, editing posts, commenting,
+    responding, endorsing, and so on), the server emits
+    an edx.team.activity_updated event.
+
+    The definition of activity that would trigger
+    this event does not include changes in team membership.
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+
+    object_link = utils.get_team_url_from_team_id(current_event['referer'], current_event['event']['team_id'])
```

Comment:
Please correct the indentation on this line.

Reply:
Done